### PR TITLE
updated german translation 

### PR DESCRIPTION
**Pull request by DukeDukeEm**

### CODE REVIEW

File: community/item-schema.json [hunk 1]
```diff
@@ -1,280 +1,338 @@
 {
```
Comment (BenLubar) [inline community/item-schema.json:1]: Can you change the formatting back to how it was? Currently your PR changes every line in the item schema, which makes it hard to tell what actually changed.

File: community/statsweb_german.vdf [hunk 2]
```diff
@@ -5,353 +5,353 @@
 {
 		"StatsWeb_SiteName"		"Alien Swarm: Reactive Drop Stats"
 		"[english]StatsWeb_SiteName"		"Alien Swarm: Reactive Drop Stats"
-		"StatsWeb_GlobalWeaponStats"		"Global Weapon Stats"
+		"StatsWeb_GlobalWeaponStats"		"Globale Waffenstatistiken"
```
Comment (BenLubar) [inline community/statsweb_german.vdf:8]: Strings that are done being translated should have the two tabs at the start of the line removed.

File: resource/reactivedrop_german.txt [hunk 3]
```diff
@@ -643,14 +643,14 @@
...
-		"asw_alert_infested"		"- INFESTED -"
-		"[english]asw_alert_infested"		"- INFESTED -"
-		"asw_alert_poisoned"		"- POSIONED -"
-		"[english]asw_alert_poisoned"		"- POSIONED -"
-		"asw_alert_reloading"		"- RELOADING -"
-		"[english]asw_alert_reloading"		"- RELOADING -"
-		"asw_alert_ffguard"		"- FRIENDLY FIRE GUARD -"
-		"[english]asw_alert_ffguard"		"- FRIENDLY FIRE GUARD -"
+"asw_alert_infested"		"- INFESTED -"
+"[english]asw_alert_infested"		"- INFESTED -"
```
Comment (BenLubar) [inline resource/reactivedrop_german.txt:647]: Strings that are not translated yet should be indented with two tabs to help them stand out.

**Review by BenLubar — CHANGES_REQUESTED** [review]

File: community/statsweb_german.vdf (hunk 2 above)
Comment (DukeDukeEm) [reply community/statsweb_german.vdf]: Ok, I did this.
Why isn't it like this in the 563560_loc_<lang>.vdf files?

**Pull request merged**